requirements-dev.txt

```diff
@@ -8,7 +8,7 @@ appdirs==1.4.4
     # via
     #   black
     #   virtualenv
-attrs==20.3.0
+attrs==21.2.0
```

Comment:
Does this change back if the developer runs `pip-compile requirements-dev.in`? We might need to consider that we inherit libraries from base requirements.txt

Reply:
It doesn't change back. Actually both `requirements.txt` and `requirements-dev.txt` in this PR were created by first running `pip-compile requirements.in` and then `pip-compile requirements-dev.in`.